packages/react-server/src/ReactFizzServer.js

```diff
@@ -305,7 +306,17 @@ export function createRequest(
     onShellError: onShellError === undefined ? noop : onShellError,
     onFatalError: onFatalError === undefined ? noop : onFatalError,
   };
-  // This segment represents the root fallback.
+  const rootBoundaryID = getRootBoundaryID(responseState);
+  let maybeRootBoundary = null;
+  if (rootBoundaryID !== UNINITIALIZED_SUSPENSE_BOUNDARY_ID) {
+    const fallbackAbortSet: Set<Task> = new Set();
+    const rootBoundary = createSuspenseBoundary(request, fallbackAbortSet);
```

Comment:
This part makes me a bit uncomfortable because it's not a Suspense boundary on the client although it kind of acts like one. We're going to be splitting Suspense boundaries into the Offscreen boundaries for the server and changing how to annotations work and how they have to line up.